cfme/scripting/quickstart.py

```diff
@@ -219,34 +212,8 @@ def disable_bytecode(venv_path):
     venv_call(venv_path, 'python', '-m', 'cfme.scripting.disable_bytecode')
 
 
-def link_config_files(src, dest):
-    if not os.path.isdir(src):
-        print("WARNING: not linking config files,", src, "missing")
-        return
-
-    bad_elements = False
-
-    for element in os.listdir(src):
-        if element.endswith(('.yaml', '.eyaml')):
-            if element.endswith('.eyaml') and not os.path.exists('.yaml_key'):
-                print(
-                    "WARNING:", element, "is encrypted, "
-                    "please remember follow the documentation on yaml keys")
-            target = os.path.join(dest, element)
-            source = os.path.relpath(os.path.join(src, element), dest)
-            # the following is fragile
-            if os.path.islink(target):
-                if os.readlink(target) != source:
-                    print("WARNING:", target, "does not point to", source)
-                    print("         please verify this is intended")
-            elif os.path.isfile(target):
-                print('ERROR: You have', element, 'copied into your conf/ folder. Remove it.')
-                bad_elements = True
-            else:
-                os.symlink(source, target)
-
-    if bad_elements:
-        exit(1)
+def link_config_files(venv_path, src, dest):
```

Comment:
**Required**
When I checkout your branch and try to run `miq shell`, this raises an exception because the call on line 234 doesn't include the new param.

Reply:
@RonnyPfannschmidt Is this fixed now?

Reply:
yes